chain/substreams/src/chain.rs

```diff
@@ -57,6 +57,10 @@ impl blockchain::Block for Block {
     fn parent_ptr(&self) -> Option<BlockPtr> {
         None
     }
+
+    fn timestamp(&self) -> BlockTime {
+        BlockTime::NONE
```

Comment:
Should this type be called something like MaybeBlockTime or something just to indicate it may be none even though it's not an option? 

I guess the substreams notion of blocks doesn't fit the aggregations model anyway

Reply:
The `NONE` thing should really go away (maybe except for tests) I just put it there for the few cases where I wasn't sure what's happening (like substreams)